java/src/test/java/ai/onnxruntime/InferenceTest.java

```diff
@@ -66,7 +68,7 @@ public class InferenceTest {
   private static final Pattern inputPBPattern = Pattern.compile("input_*.pb");
   private static final Pattern outputPBPattern = Pattern.compile("output_*.pb");
 
-  private static final OrtEnvironment env = OrtEnvironment.getEnvironment();
+  private static final OrtEnvironment env = TestHelpers.getOrtEnvironment();
```

Comment:
note: added this helper which allows one to set an environment variable to customize the OrtEnvironment log level. it was useful for debugging.